src/Makefile.common.GPU

```diff
@@ -1,39 +1,15 @@
 # CPP compiler
 CC ?= g++
 
-# for using some pre-set compiler options on specific machines
-# you can add your own here
-host_name := $(shell hostname -f)
-ifeq ($(findstring titan, $(host_name)), titan)
-    machine_name := titan
-else ifeq ($(findstring doppio, $(host_name)), doppio)
-    machine_name := doppio
-else ifeq ($(findstring rhea, $(host_name)), rhea)
-    machine_name := rhea
-endif
-
-
 # if FC is set to pgi, we use pgi compiler
 ifeq ($(FC),pgi)
-    ifeq ($(machine_name), titan)
-	CLAW_FC ?= ftn
-    else ifeq ($(machine_name), rhea)
-	CLAW_FC ?= pgfortran
-    else
-        CLAW_FC ?= pgfortran
-    endif
+    CLAW_FC ?= pgfortran
```

Comment:
Usually `FC` is set to the fortran compiler command. Here you seem to be setting it to `pgi` and then using that to set `CLAW_FC` to the compiler command.  Why not just set `FC ?= pgfortran` in the application makefile, which would be more consistent with other usage?

I see from the old version that on some machines the pgi compiler name is `ftn` but when running on such a machine the user should just set `FC` to `ftn`?

Reply:
I think I was just following what's done in regular Makefile.common where we have FC and then CLAW_FC ?= FC. Then in order to stress that you need a special GPU-enable pgi compiler, I set it to pgi to distinguish it from regular pgfortran. "ftn" is just an alias to pgfortran on one of the machine I have used. 

I can change this if that's better. I don't fully understand why do we have both FC and CLAW_FC though. Why not just FC?